src/diagnostics.rs

```diff
@@ -224,11 +224,11 @@ pub fn report_error<'tcx, 'mir>(
                     (None, format!("or pass `-Zmiri-isolation-error=warn` to configure Miri to return an error code from isolated operations (if supported for that operation) and continue with a warning")),
                 ],
             StackedBorrowsUb { help, history, .. } => {
-                let url = "https://github.com/rust-lang/unsafe-code-guidelines/blob/master/wip/stacked-borrows.md";
                 msg.extend(help.clone());
                 let mut helps = vec![
                     (None, format!("this indicates a potential bug in the program: it performed an invalid operation, but the Stacked Borrows rules it violated are still experimental")),
-                    (None, format!("see {url} for further information")),
+                    (None, format!("see https://github.com/rust-lang/unsafe-code-guidelines/blob/master/wip/stacked-borrows.md for further information")),
+                    (None, format!("please reach out to us (e.g. at <https://rust-lang.zulipchat.com/#narrow/stream/269128-miri>) if you find this troubling")),
```

Comment:
Do the `< >` have any particular meaning here? I can only think they'd make it harder to copy this into a URL bar.

Reply:
They are commonly used in email to mark URLs, so that it is easy to tell where it begins and ends and to deal with linebreaks that have been introduced.

In this case there is a `)` after the URL and I was not sure if that woukd split correctly in all terminal emulators.

Reply:
That makes sense.